src/index.ts

```diff
@@ -569,6 +563,18 @@ export default runExtension({
         ),
     });
 
+    getSamePageAPI().then((api) => {
+      api.listNotebooks().then(({ notebooks }) => {
+        registerDatalogTranslator({
+          key: "is in notebook",
+          callback: () => [],
+          isVariable: true,
+          placeholder: `Roam ${window.roamAlphaAPI.graph.name}`,
+          targetOptions: notebooks.map((n) => `${n.appName} ${n.workspace}`),
+        });
+      });
+    });
```

Comment:
with samepage enabled, Query builder will have access to a new `is in notebook` relation

Reply:
Could you remove the notebook limit on my samepage account?

Reply:
If I use `node is in notebook someNotebook`, should that  only return nodes in `someNotebook`?